client/components/overview/ProductOverview.jsx

```diff
@@ -1,83 +1,103 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
+import {FacebookShareButton, PinterestShareButton,TwitterShareButton} from 'react-share';
 import GITHUB_API_TOKEN from '../../config.js';
 import StyleSelector from './StyleSelector.jsx';
 import Rating from './Rating.jsx';
-import getProducts from './utils.jsx';
-import { Grid } from '@material-ui/core'
+import { Grid, Typography } from '@material-ui/core'
```

Comment:
https://material-ui.com/guides/minimizing-bundle-size/#when-and-how-to-use-tree-shaking